### PR TITLE
Demo Data

### PR DESCRIPTION
**Pull request by hennevogel**
Add a rake task to load the :full_conference factory into the database. We're going to use this for the demo instance and the review apps.

### CODE REVIEW

File: spec/factories/question.rb [hunk 1]
```diff
@@ -2,11 +2,9 @@
...
 FactoryGirl.define do
   factory :question do
-    title 'blah'
+    title { Faker::Lorem.sentence }
     question_type
-    after(:build) do |question|
-      question.answers << build(:answer)
-      question.conferences << build(:conference)
-    end
+    conferences { [create(:conference)] }
```
Comment (ChrisBr) [inline spec/factories/question.rb]: create_list ?

File: spec/factories/question.rb [hunk 2]
```diff
@@ -2,11 +2,9 @@
...
+    answers { [create(:answer)] }
```
Comment (ChrisBr) [inline spec/factories/question.rb]: same here

File: spec/factories/question.rb (hunk 1 above)
Comment (hennevogel) [inline spec/factories/question.rb]: What is the difference between create_list with 1 and create? :-)

**Comment by ChrisBr** [comment]
LGTM

File: spec/factories/question.rb (hunk 1 above)
Comment (ChrisBr) [inline spec/factories/question.rb]: I guess you don't have to wrap it into the array anymore?

Comment (hennevogel) [inline spec/factories/question.rb]: True. But I find this not very obvious...

Comment (ChrisBr) [inline spec/factories/question.rb]: Ok, then leave it as it is

**Pull request merged**